backend/db-setup/generate-db-data.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Is it worth being able to take in a variable value that determines how many lines to generate? Say, if we wanted to simulate a local copy of prodlike data?

Reply:
Doing this would be fine, but this file no longer exists and has been replaced by a sample SQL file that shows how to fill the faker tables. Let's revisit this request when it becomes more than an example.